src/runtime/composables/useFeatureDetection.ts

```diff
@@ -0,0 +1,13 @@
+import { onMounted } from 'vue'
+
+export function useFeatureDetection(id: string) {
+  onMounted(() => {
+    onMounted(() => {
```

Comment:
I think that might be a typo to have two onMounted() calls nested?

Reply:
:joy: oops